### ISSUE TITLE
Cut out great lakes

### ISSUE THREAD

**Issue by anandthakker** [anandthakker/us-atlas#4]
We need to fix our state boundaries to show the great lakes.

The problem is that in the we're building `geojson/states.geojson` target, we're using natural earth's `ne_110m_admin_1_states_provinces`, which doesn't have the lake boundaries.  There's already a `us/states.shp` target that makes a shapefile that _does_ show the lakes.... it's possible that reason we're not using it is that the turf stuff we're doing in `inset-polygons` was blowing upon those geometries, but I'm honestly not exactly sure.  Since we aren't using the insets anymore, this might be a quick/easy way to do it.

So, basically, this task is: fix the `geojson/states.geojson` target to make a geojson file that (a) actually shows the boundaries, and (b) if necessary change [these lines](https://github.com/anandthakker/us-atlas/blob/master/Makefile#L797-L798) to match the property name for the state postal code
